helper/src/components/clusterTab.js

```diff
@@ -69,8 +69,8 @@ export default function ({ tabValues, updateFn, featureFlag, invalidArray }) {
                         <ChoiceGroup
                             selectedKey={cluster.AksPaidSkuForSLA}
                             options={[
-                                { key: false, text: 'Free clusters with a service level objective (SLO) of 99.5%' },
-                                { key: true, text: 'Uptime SLA guarantees 99.95% availability of the Kubernetes API server endpoint for clusters that use Availability Zones' }
+                                { key: false, text: 'Free clusters with a service level objective (SLO) of 99.9%' },
```

Comment:
Free clusters offer an SLO of 99.5%, not 99.9%
![image](https://user-images.githubusercontent.com/17914476/214805383-2e098c7b-9474-4104-9739-7f8a93d2fad3.png)
>  The existing free version of AKS will continue to offer a service level objective (SLO) of 99.5% for both regional and AZ clusters 
Ref: https://techcommunity.microsoft.com/t5/apps-on-azure-blog/aks-introduces-uptime-sla/ba-p/1350832